src/docs/metrics/mean-realized-price/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Mean Realized Price
+author: Filip
+date: 2024-04-01
+description: Mean Realized Price Metrics
+---
+
+## Definition
+
+The Mean Realized Price metric calculates the average price at which tokens have been transacted 
+relative to their current circulating supply. It provides insights into the average profitability 
+of token transactions by considering the [realized value](/metrics/realized-value) against the 
```

Comment:
```suggestion
of token transactions by considering the [Realized Value](/metrics/realized-value) against the 
```
Let's keep the metric naming convention consistent. We have `Mean Realized Price` above.

Reply:
Fixed